addons/mail/static/tests/message/link_preview.test.js

```diff
@@ -364,13 +406,16 @@ test("Delete all link previews at once", async () => {
         },
     ]);
     const channelId = pyEnv["discuss.channel"].create({ name: "wololo" });
-    pyEnv["mail.message"].create({
+    const messageId = pyEnv["mail.message"].create({
         body: "not empty",
-        link_preview_ids: linkPreviewIds,
         message_type: "comment",
         model: "discuss.channel",
         res_id: channelId,
     });
+    pyEnv["mail.link.preview.message"].create([
+        { message_id: messageId, link_preview_id: linkPreviewIds[0] },
+        { message_id: messageId, link_preview_id: linkPreviewIds[1] },
+    ]);
```

Comment:
Nitpick: you destruct to `linkPreviewId_1` & `linkPreviewId_2` in `pyEnv["mail.link.preview"].create()`.

This doesn't seem much but gives a bit more confidence that the link previews exist. If we add or remove some link preview, it's more prone to forget to adapt `[0]`/`[1]` here than with the destruct to their own `const` per link preview.